### PR TITLE
removed MacOS from intro

### PR DESCRIPTION
**Pull request by kevinnls**
- [x] I solemnly swear that this is ~~all original content of which I am the original author~~ a change to the original that is factually accurate
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] ~~If you've~~ I have not changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

MacOS no longer ships bash as the default shell, so I felt it necessary to remove it from the intro.

### CODE REVIEW

**Comment by kevinnls** [comment]
I am sorry; the New Issue template was not very clear as to whether an issue has to be created or not for minor changes.
But I went ahead and made this PR without an issue. My apologies if this was wrong. 
And I request that the preference (be it whichever) be specified in the template.

File: bash.html.markdown [hunk 1]
```diff
@@ -23,7 +23,7 @@ translators:
 ---
 
 Bash is a name of the unix shell, which was also distributed as the shell
-for the GNU operating system and as default shell on Linux and Mac OS X.
```
Comment (mxschumacher) [inline bash.html.markdown:26]: I don't understand the change, isn't bash also the default shell on macOS?

Comment (gfting) [reply bash.html.markdown:26]: zsh has become the default shell: https://support.apple.com/en-us/HT208050

Comment (mxschumacher) [reply bash.html.markdown:26]: ah okay, thank you. From Catalina onward I see.

Comment (kevinnls) [reply bash.html.markdown:26]: thank you @gfting i couldn't remember where i had seen it!

**Pull request merged**